certify-mock-identity.properties

```diff
@@ -29,7 +29,7 @@ mosip.certify.database.name=inji_certify_mock
 # conditionally set the SOFTHSM pin based on the setup requirements and fallback to the latter if former value is missing
 mosip.kernel.keymanager.hsm.keystore-pass=${softhsm.certify.mock.security.pin:${softhsm.certify.security.pin}}
 # Set the key prefix for Redis cache
-spring.cache.redis.key-prefix=${mosip.injicertify.mock.host}::
+mosip.certify.cache.redis.prefix=${mosip.injicertify.mock.host}::
```

Comment:
mosip.certify.cache.redis.prefix lets change to mosip.certify.cache.redis.key-prefix